src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/55.diff.sql

```diff
@@ -0,0 +1,133 @@
+﻿SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+GO
+
+IF NOT EXISTS 
+(
+    SELECT *
+    FROM    sys.indexes
+    WHERE   NAME = 'IXC_FileProperty_InstanceKey_Watermark_ContentLength'
+        AND Object_id = OBJECT_ID('dbo.FileProperty')
+)
+BEGIN
+    CREATE NONCLUSTERED INDEX IXC_FileProperty_InstanceKey_Watermark_ContentLength ON dbo.FileProperty
```

Comment:
Do not put creating index inside the transaction, it will lock the whole table until the duration of the transaction. If it s a bigger table, the migration will get affected